bwrap.xml

```diff
@@ -190,10 +190,21 @@
   <para>
     Filesystem related options. These are all operations that modify the filesystem directly, or
     mounts stuff in the filesystem. These are applied in the order they are given as arguments.
+  </para>
+  <para>
     Any missing parent directories that are required to create a specified destination are
-    automatically created as needed.
+    automatically created as needed. Their permissions are normally set to 0755
+    (rwxr-xr-x). However, if a <option>--perms</option> option is in effect, and
+    it sets the permissions for group or other to zero, then parent directories
+    will also have their permissions set to zero.
```

Comment:
This is not actually 100% consistent with the current implementation, but it *is* consistent with the implementation I'm now proposing based on your review!

Might be clearer as

```suggestion
    will also have the corresponding permission set to zero.
```